k8s-deploy/development/deployment_sidekiq.yaml

```diff
@@ -21,6 +21,7 @@ spec:
     spec:
       containers:
         - name: parliamentary-questions-rails-jobs
+          image: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/pq-team/parliamentary-questions:development.latest
```

Comment:
Not sure whether this is required as there were not there originally. However I think its relevant to specify the image for each env